ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/DecoderOptions.java

```diff
@@ -14,11 +14,12 @@
  */
 public interface DecoderOptions extends SinkOptions, PipelineOptions {
 
-  @Description("Static IP associated with load balancer; will be ignored for IP lookups")
-  @Default.String("35.227.207.240")
-  String getProxyIpAddress();
+  @Description("Comma separated list of static IPs associated with load balancer; "
+      + "will be ignored for IP lookups")
+  @Default.String("35.227.207.240,34.120.208.123")
+  String getProxyIpAddressCsv();
```

Comment:
I recommend `proxyIps` per https://github.com/mozilla-services/cloudops-infra/pull/3582/files#diff-8c0bf74f03d3c2a36089bf3c606bf05cbc746f13f5ad989f72341f28dfa91915R48 or perhaps `proxyIpAddresses`. We have a couple of CSV parameters and none are explicitly named such, so I would want to stick to avoid suffixing with `CSV` here.